embuilder.py

```diff
@@ -173,12 +174,15 @@ def main():
                       help='show build commands')
   parser.add_argument('--wasm64', action='store_true',
                       help='use wasm64 architecture')
-  parser.add_argument('operation', help='currently only "build" and "clear" are supported')
-  parser.add_argument('targets', nargs='+', help='see below')
+  parser.add_argument('operation', choices=['build', 'clear', 'rebuild'])
+  parser.add_argument('targets', nargs='*', help='see below')
```

Comment:
Do you know why you changed this from `+` to `*`?

If seems that it caused a regression in behaviour: https://github.com/emscripten-core/emscripten/issues/21466